modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -0,0 +1,132 @@
+[id="proc-configuring-postgresql-instance-using-helm_{context}"]
+= Configuring an external PostgreSQL instance using the Helm Chart
+
+You can configure an external PostgreSQL instance by using the Helm Chart. By default, the Helm Chart creates and manages a local instance of PostgreSQL in the same namespace where you have deployed the {product-very-short} instance. However, you can change this default setting to configure an external PostgreSQL database server, for example, AWS RDS or Azure database. 
+
+.Prerequisites
+
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+
+* You have a PostgreSQL certificate of the .pem format so that you can secure your database connection using the Secure Socket Layer (SSL) or Transport Layer Security (TLS) protocol. For more information, see link:{LinkAdminGuide}#postgresql-certificate-for-aws-rds_admin-rhdh[Downloading PostgreSQL certificate for AWS RDS] or link:{LinkAdminGuide}#generating-postgreSQL-certificate-for-azure-database_admin-rhdh[Generating PostgreSQL certificate for Azure database].
```

Comment:
Both links are broken: [Downloading PostgreSQL certificate for AWS RDS](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.3/html-single/administration_guide_for_red_hat_developer_hub/index#postgresql-certificate-for-aws-rds_admin-rhdh) or [Generating PostgreSQL certificate for Azure database](https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.3/html-single/administration_guide_for_red_hat_developer_hub/index#generating-postgreSQL-certificate-for-azure-database_admin-rhdh).

Reply:
Because it points to RHDH 1.2 docs, which is still in progress. Once we publish the 1.2 doc, those links should work.